### PR TITLE
Update readme add final models

### PR DESCRIPTION
**Pull request by ramaneswaran**
I should have preferably broken this down into smaller PRs

### CODE REVIEW

**Comment by L04DB4L4NC3R** [comment]
@Geek-ubaid

File: app/run.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+from song_story import GeniusClient
+
+load_dotenv()
+
+
+class App(Flask):
+    def __init__(self, name):
+        super(App, self).__init__(name)
+
+        # Load models
```
Comment (Geek-ubaid) [inline app/run.py]: Don't include so much comments in between, anything which is straightforward can be understood with name of the variable or function used

File: app/test.py [hunk 2]
```diff
@@ -1,40 +1,56 @@
 import spacy
+import os
 import torch
-from core.nlu.spacy import SpacyModel
-from core.summarizer.textrank import TextRank
-from core.language.gpt2 import GPT2
+from song_story import SpacyModel
+from song_story import TextRank
```
Comment (Geek-ubaid) [inline app/test.py]: shift tests file to test folder

File: run.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
+import os
+import re
```
Comment (Geek-ubaid) [inline run.py:2]: why is this run.py file here, there is one more in app folder, remove it if pushed by mistake

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    "1 - Planning",
+    "2 - Pre-Alpha",
+    "3 - Alpha",
+    "4 - Beta",
+    "5 - Production/Stable",
+    "6 - Mature",
+    "7 - Inactive",
+]
+py_versions = (
+    "2.0 2.1 2.2 2.3 2.4 2.5 2.6 2.7 3.0 3.1 3.2 3.3 3.4 3.5 3.6 3.7 3.8".split()
```
Comment (Geek-ubaid) [inline setup.py:34]: you can simply write python 2 and python 3 versions like >=python3.0 etc

File: song_story/core/nlu/spacy.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    """
+
+    def __init__(self, size="md"):
+
+        # Model size
+        if size in ["md", "sm", "lg"]:
+            model_size = "en_core_web_{}".format(size)
+        # Load NLP model
+        self.nlp = spacy.load(model_size, disable=["tagger", "ner"])
+        # Load Clist
```
Comment (Geek-ubaid) [inline song_story/core/nlu/spacy.py]: remove commented code if they are of no use

File: song_story/prompt-generation/utils.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+def encode(batch, tokenizer, max_len):
+    """
+    This function takes in a batch and a tokenizer and outputs encoded batch
+    """
```
Comment (Geek-ubaid) [inline song_story/prompt-generation/utils.py:4]: why to use a new file when you have no imports and a function

File: tests/test_hello.py [hunk 7]
```diff
@@ -0,0 +1,3 @@
+def test_simple():
+    print("Hello World")
```
Comment (Geek-ubaid) [inline tests/test_hello.py]: remove this file

**Review by Geek-ubaid — CHANGES_REQUESTED** [review]

**Comment by Geek-ubaid** [comment]
@ramaneswaran note all the changes requested and why is second PR opened if this is not merged ?

**Comment by ramaneswaran** [comment]
> @ramaneswaran note all the changes requested and why is second PR opened if this is not merged ?

That one is for the web branch, it has smaller models

**Comment by Geek-ubaid** [comment]
@ramaneswaran changes are still not made, please make the changes requested so that i can merge the PR

File: app/run.py (hunk 1 above)
Comment (ramaneswaran) [reply app/run.py]: Removed the comments

File: app/test.py (hunk 2 above)
Comment (ramaneswaran) [reply app/test.py]: Removed the test.py, it was actually being used to demo the functions while developing, should have used different name

File: run.py (hunk 3 above)
Comment (ramaneswaran) [reply run.py:2]: The app folder is now removed, it was causing errors with imports so run.py is now in root dir

File: song_story/core/nlu/spacy.py (hunk 5 above)
Comment (ramaneswaran) [reply song_story/core/nlu/spacy.py]: Removed the code

File: song_story/prompt-generation/utils.py (hunk 6 above)
Comment (ramaneswaran) [reply song_story/prompt-generation/utils.py:4]: This function is being imported into two different files for training and inference, hence created a separate file for it

File: tests/test_hello.py (hunk 7 above)
Comment (ramaneswaran) [reply tests/test_hello.py]: Removed it

**Comment by ramaneswaran** [comment]
@Geek-ubaid  I have made the changes

File: setup.py (hunk 4 above)
Comment (ramaneswaran) [reply setup.py:34]: Removed this, it was redundant

**Comment by L04DB4L4NC3R** [comment]
> @Geek-ubaid I have made the changes

@Geek-ubaid @SaurusXI

**Review by Geek-ubaid — APPROVED** [review]

**Pull request merged**